### DISCUSSION TITLE
Global.css style not applying/working on pages/components

### DISCUSSION THREAD

**Discussion by yelnyafacee** [vercel/next.js#26820] [Help]
### What version of Next.js are you using?

v10.0.9

### What version of Node.js are you using?

v14.16.0

### What browser are you using?

Chrome

### What operating system are you using?

Windows 10

### How are you deploying your application?

Localhost

### Describe the Bug

Global.css style not applying/working on pages/components even after imported in `_app.js`, the styles are not applied to pages/component even when it is explicitly used in the HTML tags

### Expected Behavior

CSS classes/style from global.css applied to all pages/component using it

### To Reproduce

1. Create _app.js
2. add `import '../styles/globals.css'` to `_app.js`
3. Create a page
4. Create the page JSX that use a css class style from `global.css`

I am new to `nextjs`

I have a `css` class in `styles/globals.css`:

```
.section-sidebar-filter-name {
  margin: 0 0 .5rem;
}
```

and I have the `import` statement in `pages/_app.js`:

```
import '../styles/globals.css'

function MyApp({ Component, pageProps }) {
...

};

```

but the `CSS` class style is not applied to the `react components` in the `pages` when I view it in chrome browser developer mode ( `Example: test.js page `):

```
function Test() {

   return(
      <h3 className="c-clickable section-sidebar-filter-name">Test</h3>
   )

}

```

next.config.js:

```
module.exports = {
    webpack: (config) => {
        const rules = config.module.rules
            .find((rule) => typeof rule.oneOf === 'object')
            .oneOf.filter((rule) => Array.isArray(rule.use));

        rules.forEach((rule) => {
            rule.use.forEach((moduleLoader) => {
                if (moduleLoader.loader.includes('css-loader') && !moduleLoader.loader.includes('postcss-loader')) {
                    delete moduleLoader.options.modules.getLocalIdent;
                    moduleLoader.options = {
                        ...moduleLoader.options,
                        modules: {
                            ...moduleLoader.options.modules,
                            localIdentName: '[folder]__[local]--[hash:base64:5]',
                            // getLocalIdent: (context, localIdentName, localName, options) => {
                            //     return "whatever_random_class_name";
                            // },
                            // You can also add other css-loader options here
                        }
                    };
                }
            });
        });

        return config;
    }
};
```

anyone knows what I did wrong?

**Comment by diegomarcuz** [comment]
When I am using dynamic routes and enter the app by a specific route (not from home), styles are not being applied as well.

**Comment by arcataroger** [comment]
Having the same problem... the CSS selectors never even seem to get added to the page's `<style>` tag. I can't tell if this is some sort of tree-shaking going on...

**Comment by santosh3700** [comment]
same issue with me. Class base styling not working. While tag base styling is working

**Comment by xtinktstudios** [comment]
Same here, any page_name.js added to ./pages isn't styles by the globals.css and I cannot, for the life of me, figure out how to make it apply

**Comment by dev-athar** [comment]
hey good ppl!
the globals.css does not works for some reason.
here a workaround [It should work], add to pages>index.js. 
> import "../app/globals.css"
